### ISSUE TITLE
How to record THE BAG in MID100

### ISSUE THREAD

**Issue by sczgzzzj** [Livox-SDK/livox_mapping#4]
How do I modify the livox_lidar.launch file？or need to modify other files？thanks

**Comment by Livox-SDK** [comment]
Hi, sczgzzzj. Do you mean running livox_mapping using your own recorded Mid 100 bag?

**Comment by sczgzzzj** [comment]
Yes, a self-recorded bag cannot be counted，How to record the correct bag with MID100，tanks

**Comment by Livox-SDK** [comment]
Hi sczgzzzj,
sorry since mid-100 is an old-version product, we haven't tested it under mid-100 
I think It should be alright If the data from mid-100 is stored in the same bag.
